node.js

```diff
@@ -1 +1,116 @@
-throw new Error('JWT Client does not support Node.js yet. PR welcome.');
+global.localStorage = {};
+global.localStorage.setItem = function(item, value){
+    global.localStorage[item] = value;
+};
+global.localStorage.getItem = function(item){
+    return global.localStorage[item];
+};
+global.localStorage.removeItem = function(item){
+    delete global.localStorage[item];
+};
```

Comment:
Wouldn't it be better to use a plain object inside the function clojure rather than puting it inside the `global` object?